test/cancun/kzg-point-evaluation/KZGPointEvaluation.js

```diff
@@ -0,0 +1,106 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const { expect } = require('chai');
+const { ethers } = require('hardhat');
+const Constants = require('../../constants');
+
+describe('@cancun KZG Point Evaluation Test Suite', () => {
+  let kzgPointEvalContract;
+
+  before(async () => {
+    const kzgPointEvalFac = await ethers.getContractFactory(
+      Constants.Contract.KZGPointEvaluation
+    );
+    kzgPointEvalContract = await kzgPointEvalFac.deploy();
+  });
+
+  it('Should successfully execute evaluateKZGProof() with valid parameters', async () => {
+    // example input and expected output from https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/resource/contract/contracts/Module050OpcodesExist/Module050OpcodesExist.sol#L34C34-L34C418
```

Comment:
TY for the comment detailing where this magic input constant - which has to be _just right_! - came from...

Reply:
out of my curiosity how did you compute that input?